examples/python/objectron/README.md

```diff
@@ -3,7 +3,7 @@ title = "Objectron"
 tags = ["2D", "3D", "Object detection", "Pinhole camera", "Blueprint"]
 thumbnail = "https://static.rerun.io/objectron/b645ef3c8eff33fbeaefa6d37e0f9711be15b202/480w.png"
 thumbnail_dimensions = [480, 480]
-# channel = "release"  - Disabled because it sometimes have bad first-frame heuristics
+# Channel = "release" - disabled because it sometimes have bad first-frame heuristics
```

Comment:
this is an unfortunate change, but I kept it because opting out here is too much hassle

Reply:
I think the correct way to handle this is to ignore everything within `<!--` and the next `-->`, but I can understand if this was too much of a hassle right now

Reply:
yeah the problem is that we have another check that is responsible for having a look at this metadata...